### DISCUSSION TITLE
GrapesJS Storage Manager Docs Should Be Updated to Reflect Changes Made in 0.19.4

### DISCUSSION THREAD

**Discussion by Drew-Daniels** [GrapesJS/grapesjs#4571] [Ideas]
On the [Getting Started Tutorial](https://grapesjs.com/docs/getting-started.html), this example configuration is shown under the [Store & Load Data Section](https://grapesjs.com/docs/getting-started.html#store-load-data), where `storeComponents`, `storeStyles`, `storeHtml`, and `storeCss` are shown as optional configurations:

![Pasted image 20220908083750](https://user-images.githubusercontent.com/62859552/189219266-41d535b8-fcad-4a33-be75-560434df76de.png)

However, in `0.19.4`, these storage options were removed, as mentioned in the [Release Notes](https://github.com/artf/grapesjs/releases):

![Pasted image 20220908083808](https://user-images.githubusercontent.com/62859552/189219335-f981423b-2375-46c0-8f4c-91105b3817cf.png)

I do not think GrapesJS should encourage users to follow these options given they are no longer supported.

**Comment by artf** [comment]
Thanks Drew, you're right, I'll update it.

**Reply by clonefunnels** [reply]
So just to make sure I'm clear... Is this supposed to still work or not?

![2022-12-18_05h47_17](https://user-images.githubusercontent.com/95948977/208294093-6d4796d2-280f-45f1-8dd0-8441867929f6.png)

I get a brown screen of death when I try it.

**Reply by clonefunnels** [reply]
Would it be like this:

          headers: {header('Content-Type: application/json')}, // Custom headers for the remote storage request
         
Or like this:  
 headers: {'Content-Type: application/json'}, // Custom headers for the remote storage request

Or this:

 headers: {application/json}, // Custom headers for the remote storage request

Or something else?

It would be great if you could give some real life examples in your tutorials.

**Reply by clonefunnels** [reply]
These are my headers that work with old version.

$response = [
'html' => $html,
'css' => $css,
];
header('Content-Type: application/json');
echo json_encode($response);

**Reply by artf** [reply]
sorry @clonefunnels I don't want to be rude, but maybe you should learn at least the basics of JS before diving into something like GrapesJS otherwise you'll continuously hit the wall and will pollute the discussion with questions completely out of scope of this repository.
